### DISCUSSION TITLE
Installation/ test

### DISCUSSION THREAD

**Discussion by scote89** [JWock82/Pynite#121] [General]
Hi, Im curious to try your examples out but there isnt much information on installation.

After running pip install PyNiteFEA, I get No module named 'PyNite' when trying your examples out. What am I doing wrong?

**Comment by JWock82** [comment]
This usually has to do with your PYTHONPATH variable not including your module installation folder. What program are you running python from? Make sure your PYTHONPATH has that folder listed in it.

**Comment by scote89** [comment]
ok so I managed to install pynite correctly but am getting the following error

`ImportError: cannot import name 'FEModel3D' from 'Pynite' (unknown location)`

I can see the FEModel3D.py file in my pynite path but its somehow is still mixed up...

![image](https://user-images.githubusercontent.com/50223503/159924007-28f0786a-e709-4df6-8e3b-27ad5c6a06b5.png)


Is it something to do with me installing pynite AND pyniteFEA? Which one should I install?

Thanks for your help!

**Reply by JWock82** [reply]
Uninstall `pynite` and keep your installation of `PyNiteFEA`. Then try again.

See the link below for clarification:
https://github.com/JWock82/PyNite/wiki/1.-Getting-Started
